fabfile/alba.py

```diff
@@ -356,34 +509,50 @@ def is_local(osd):
             return True
         return False
 
-    def inner():
+    def inner(abm_cfg):
+        if _use_tls(tls):
+            abm_cfg = TLS['arakoon_config_file'] #TODO:run_tests_recovery ?
```

Comment:
also you may want to introduce a method `get_default_arakoon_cfg(tls)` and use that everywhere